llamacpp/Quantization.md

```diff
@@ -0,0 +1,73 @@
+# Quantization Documentation
+This document provides explanations for quantization methods from [llamacpp](https://github.com/ggerganov/llama.cpp), though they have not yet been verified by the original authors.
+
+### To have a brief understanding of what quantization is of tu fresh up you knowledge consider this take a look at this resources:
```

Comment:
This actually is quite helpful. I really only have two suggestions.

* Change the header (line 4, right above this comment), to something really simple, like

```
## Introduction to Quantization

If you are comfortable with LLMs but need to get started on quantization, we recommend the following resources to begin:

* ...
* ...
```
* before that section, i.e. in the main section lines 2-3, put in something about what you expect people to know, like (change it):
```
This document assumes you are familiar with quantization, how it works, the strengths and shortcomings, and the basics of quantization mathematics.
```

Maybe add if there is anything more detailed they show know, like what math, etc. Just set the expectations. Before someone begins, ensure they won't get lost.